google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/CloudSdkAppEngineHelper.java

```diff
@@ -149,7 +151,7 @@ public CancellableRunnable createDeployRunner(
 
     boolean isFlexCompat = targetEnvironment.isFlexible()
         && AppEngineProjectService.getInstance().isFlexCompat(project, source);
-    if (targetEnvironment.isStandard() || isFlexCompat) {
```

Comment:
why did you remove the isFlexCompat? dont we need to use the standard runner for compat projects too still?